ui/admin/tests/acceptance/host-catalogs/hosts/create-test.js

```diff
@@ -129,6 +129,8 @@ module('Acceptance | host-catalogs | hosts | create', function (hooks) {
     assert.strictEqual(getHostCount(), count);
   });
 
+  // TODO: the field should change to address and not name. Name is optional for host but address
```

Comment:
This will be solve shortly in a separate PR, despite is a small change, I think it does not belong to the specific scope of this PR.